__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const { topicData, articleData, userData, commentData } = require('../db/data/test-data/index');
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection');
+const request = require('supertest');
+
+beforeEach(() => {
+    return seed({ topicData, userData, articleData, commentData });
+});
+
+afterAll(() => {
+    return db.end();
+});
+
+describe('GET /api/topics', () => {
+    test('returns array of objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            const topics = JSON.parse(response.text).topics;
+            
+            expect(Array.isArray(topics)).toBe(true);
+
+            for(topic of topics) {
+                expect(typeof topic).toBe('object');
+            }
```

Comment:
Nice, but this is made redundant with the following test. We can have a single test of simply what we expect as a response from this endpoint